src/js/events/MarathonActions.js

```diff
@@ -492,9 +492,13 @@ var MarathonActions = {
   },
 
   killPodInstances(pod, instanceIDs, force) {
-    let podID = encodeURIComponent(pod.getId());
+    let podID = pod.getId();
     let params = '';
 
+    if (podID[0] === '/') {
+      podID = podID.substr(1);
```

Comment:
I would rather choose a regular expression to replace the `/` from the beginning of a string
```JS
podID = podID.replace(/^\//,'');
```

Or do the replace in line 495 
```JS
let podID = pod.getId().replace(/^\//,'');